src/main/java/graphql/language/AbstractNode.java

```diff
@@ -13,11 +13,13 @@
 
     private final SourceLocation sourceLocation;
     private final List<Comment> comments;
+    private final IgnoredChars ignoredChars;
 
-    public AbstractNode(SourceLocation sourceLocation, List<Comment> comments) {
+    public AbstractNode(SourceLocation sourceLocation, List<Comment> comments, IgnoredChars ignoredChars) {
         this.sourceLocation = sourceLocation;
         Assert.assertNotNull(comments, "comments can't be null");
         this.comments = new ArrayList<>(comments);
+        this.ignoredChars = Assert.assertNotNull(ignoredChars, "ignoredChars can't be null");
```

Comment:
personally I dont think these messages on asserts add value.  If it DID blow up - it was would obvious that this line is offending.  There is no insight to be ahd I think.

Anyway not a huge problem but I would just do `assertNotNull(someField)`